### PR TITLE
fix: search events and support pages, add breadcrumbs LINK-1747

### PR DESCRIPTION
**Pull request by mikkojamG**
## Description :sparkles:

Adding breadcrumbs on search and support pages.

### Closes :no_good_woman:

**[LINK-1747](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1747):**

## Screenshots :camera_flash:
Screenshot 2024-01-19 at 12 59 01

Screenshot 2024-01-19 at 12 59 12

## Additional notes :spiral_notepad:

[LINK-1747]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jorilindell — APPROVED** [review]
Looks very good. There are some commented tests for Swedish versions which could be removed totally if you like. But it's up to you.

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 1]
```diff
@@ -153,16 +153,17 @@ const generalInstructionCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/instructions/general',
+  //     keywords: '',
+  //     pageTitle: 'Allmänt',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 2]
```diff
@@ -379,16 +382,17 @@ const generalTechnologyCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/technology/general',
+  //     keywords: '',
+  //     pageTitle: 'Allmänt',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 3]
```diff
@@ -423,16 +427,17 @@ const apiCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/technology/api',
+  //     keywords: '',
+  //     pageTitle: 'API',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 4]
```diff
@@ -467,16 +472,17 @@ const imageRightsCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/technology/image-rights',
+  //     keywords: '',
+  //     pageTitle: 'Bildrättigheter',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 5]
```diff
@@ -513,16 +519,17 @@ const sourceCodeCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/technology/source-code',
+  //     keywords: '',
+  //     pageTitle: 'Källkod',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 6]
```diff
@@ -604,16 +612,17 @@ const termsOfUseCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: 'Dataskydd och användarvillkor för Linked Events.',
+  //     expectedRoute: '/sv/help/support/terms-of-use',
+  //     keywords: '',
+  //     pageTitle: 'Dataskydd och användarvillkor',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

File: src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx [hunk 7]
```diff
@@ -736,16 +745,17 @@ const featuresCases: [Language, PageValues][] = [
...
+  // [
+  //   'sv',
+  //   {
+  //     description: '',
+  //     expectedRoute: '/sv/help/features',
+  //     keywords: '',
+  //     pageTitle: 'Tjänstens egenskaper',
+  //     title: '',
+  //   },
+  // ],
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply src/domain/app/routes/helpPageRoutes/__tests__/HelpPageRoutes.test.tsx]: Removed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/317/commits/cbf200d0e7652f9ef295c87982e9a71c4c84b9ba

**Pull request merged**